source/_posts/2015-09-03-hiring-apprentices.markdown

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Hiring Apprentices"
+date: 2015-09-09 14:44:39 -0600
+comments: true
+categories: hiring, apprenticeship, mentoring, ruby
+---
+
+The [MX](http://mx.com/) apprenticeship has been a lot of work and fun so far.
+This blog post is the beginning of an effort to document what I've learned along the way.
+
+# The Goal
+
+Initially I got interested in the idea of an apprenticeship while listening to the [ruby rogues episode with Joe Mastey and Jill Lynch](https://devchat.tv/ruby-rogues/190-rr-apprenticeship-with-joseph-mastey-and-jill-lynch-of-enova).
+I help to organize a [local ruby meetup group](http://utruby.org/#uv.rb) so I liked the idea of helping new engineers.
+I also love my job at MX so I liked the idea of helping to grow our team.
+
+When I discussed the idea of an apprenticeship with people at MX, the first thing I had to do was nail down what our objective was. "Helping new engineers" wasn't going to cut it.
+
+We decided that the primary objective was to __add amazing engineers to the team__.
+
+# Amazing Apprentices
+
+Based on my experience in the local Ruby community I believed that there were some really amazing programmers who were self-taught or graduating from bootcamps that were struggling to find first jobs.
+
+The shortage of senior engineers at pretty much every company has made it really difficult to get a job as a junior developer. Lots of companies think that they can't afford to hire junior engineers because they don't have enough senior people to train them.
+There are also serious bias issues throughout the software industry and especially in the hiring processes.
```

Comment:
As a suggestion, it may be better to say, "There is a serious bias against [insert the object here] throughout..."

I think this sentence can be a very strong one. We just need to make it a little more clear.